ext/json/json.stub.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+/** @return string|false */
+function json_encode($value, int $options = 0, int $depth = 512) {}
+
+function json_decode(string $json, bool $assoc = false, int $depth = 512, int $options = 0) {}
```

Comment:
Could you please add an explicit `/** @return mixed */` here, so it's clear that the missing return type is intentionally?

I think the second argument here should be `?bool $assoc = null`. Using `?bool $assoc = false` wouldn't be wrong either, though the former matches the implementation more closely (the null means "use whatever is specified in $options, which would be non-assoc by default).

Reply:
Thanks for the explanation :+1: